### PR TITLE
Add some weekly development reports

### PR DESCRIPTION
**Pull request by avsm**
These reports have been generated from GitHub activity, with some
hand editing in the style of the LinuxKit reports. They can be
browsed directly within the repository and cover DataKit and
immediately related repositories such as ocaml-git and 9p.  Fixes
can be contributed directly to this repository.

For the brave who want to generate them for their own repositories,
the work-in-progress generator of the skeleton of these reports
is in https://github.com/avsm/hesternus

cc @samoht @talex5 @yomimono @yallop @dsheets @djs55 @justincormack @thaJeztah

### CODE REVIEW

File: reports/2017-04-17.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Since the Docker Hub doesn't support multi-stage container builds yet, the Dockerfile
+  used for autobuilds was inlined to ensure that the published images stay in sync. ([moby/datakit#521] [@talex5]).
+- The root `opam` file is now called `datakit.opam` to make it fit with the other sub-packages ([moby/datakit#525] [@talex5]).
+- The repositories and self-ci example were update for new repository locations ([moby/datakit#528] and [moby/datakit#520] [@talex5]).
+- Windows CI of DataKit was fixed up to account for recent changes ([moby/datakit#527] [moby/datakit#122] [@talex5] [@samoht]).
+
+**Functionality improvements:**
+- The GitHub bridge can now read its private key using Docker Swarm secret management, which avoids the need to spread the secret authentication token any further than necessary ([moby/datakit#519] [@talex5] [@avsm]).
+- The DataKit server no longer exposes an HTTP server. It was only used for debugging before, and Irmin 1.0 no longer supports it ([moby/datakit#524] [@talex5]).
```
Comment (talex5) [inline reports/2017-04-17.md]: I merged this, but it was @samoth's PR.

Comment (avsm) [inline reports/2017-04-17.md]: ah yes, I need to extend hesterna add the PR originator to the @ list

Comment (avsm) [inline reports/2017-04-17.md]: fixed in aebf7fdc1eb7001213b1b5d6a53c1d6b63ce1aa2

**Comment by samoht** [comment]
That looks great! Thanks you for pulling these.

**Pull request merged**